src/scripts/install-php.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+sudo apt update && sudo apt install -y software-properties-common
```

Comment:
We should probably be checking for sudo but im happy for that to be a different pr.